pkg/reconciler/route/default_config_logger.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package route
+
+import "io"
+
+// DefaultConfigLogger is a basic logger used in tests when a *testing.T type
+// is not available (such as in Examples)
+type DefaultConfigLogger struct {
+	w io.Writer
+}
+
+func (l *DefaultConfigLogger) Infof(format string, v ...interface{}) {
+	l.w.Write([]byte{})
```

Comment:
I think these should probably be `fmt.Fprintf(l.w, format, v...)` calls instead

Reply:
updated